shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -158,9 +161,13 @@ @implementation FlutterViewController {
   MouseState _mouseState;
 }
 
+// Synthesize properties with an overridden getter/setter.
 @synthesize viewOpaque = _viewOpaque;
 @synthesize displayingFlutterUI = _displayingFlutterUI;
-@synthesize prefersStatusBarHidden = _flutterPrefersStatusBarHidden;
```

Comment:
This isn't the end of the world, but typically synthesize is used to pick a new backing ivar for a property we declared ourselves, not to override superclass behaviour.